tests/unit/s2n_client_supported_versions_extension_test.c

```diff
@@ -58,6 +58,58 @@ int main(int argc, char **argv)
     struct s2n_config *config;
     EXPECT_NOT_NULL(config = s2n_config_new());
 
+    const struct s2n_security_policy *security_policy_with_tls13_and_earlier = &security_policy_20190801;
+    EXPECT_TRUE(s2n_security_policy_supports_tls13(security_policy_with_tls13_and_earlier));
+    EXPECT_EQUAL(security_policy_with_tls13_and_earlier->minimum_protocol_version, S2N_TLS10);
+
+    /* Client offers all supported versions in version list */
+    {
+        struct s2n_connection *conn = s2n_connection_new(S2N_CLIENT);
+        EXPECT_NOT_NULL(conn);
+        conn->security_policy_override = security_policy_with_tls13_and_earlier;
+
+        struct s2n_stuffer extension = { 0 };
+        EXPECT_SUCCESS(s2n_stuffer_growable_alloc(&extension, 0));
+        EXPECT_SUCCESS(s2n_client_supported_versions_extension.send(conn, &extension));
+
+        /* Check extension contains enough versions */
+        uint8_t version_list_size = 0;
+        size_t supported_versions = latest_version - S2N_TLS10 + 1;
```

Comment:
what is this math doing 🙃 please help me understand

Reply:
Its getting the total number of tls version we offer.

`34-31 + 1` = 4

```
S2N_TLS10 31
S2N_TLS11 32
S2N_TLS12 
S2N_TLS13 34
```